index.html

```diff
@@ -1,10 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <title>Tech It Easy Dashboard</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="main.js"></script>
```

Comment:
Is het een bewuste keuze dat deze bovenaan de pagina staat?
Zelf doe ik dat ook, maar gebruik ik 'defer' (of 'async') voor het laden.